webapp/src/components/github_repo_selector/github_repo_selector.jsx

```diff
@@ -51,7 +54,7 @@ export default class GithubRepoSelector extends PureComponent {
     }
 
     render() {
-        const orgOptions = this.props.yourOrgs.map((item, i) => (i === 0 ? {value: '', label: item.login} : {value: item.login, label: item.login}));
+        const orgOptions = this.props.yourOrgs.map((item) => (item.isLoggedInUser ? {value: '', label: item.login} : {value: item.login, label: item.login}));
```

Comment:
What's the reasoning behind having the `value` be blank here?

Reply:
Because this `value` is passed to the `getReposByOrg` action, if the server-side identifies an empty `organization` query parameter and no organization is specified in the config settings, it will send repositories that are owned by the logged-in user

Reply:
Why not just use the logged in user's username instead of blank string though? Just a bit harder to read, since you had to explain it to me, and still seems a bit confusing.

Reply:
Yes, it actually makes sense to pass the username directly as it would avoid unnecessary handling at the front end. Initially, the thought was to avoid calling the endpoint like this: `repos_by_org?organization=<username>`.
will do the chages

Reply:
Done.